DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -577,30 +585,30 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
 
     // MARK: - UI Events handling
 
-    @available(iOS 17.0, *)
+    @available(iOS 18.0, *)
```

Comment:
This may seem confusing, but the VPN tips feature was initially meant for iOS 17, but due to some unexpected crashes that seem to be on Apple, we've decided to move them to iOS 18.

This just needed to be updated.